guest/internal/proto/proto.go

```diff
@@ -54,12 +54,34 @@ func GetResourceVersion[O object](o O) string {
 	return ""
 }
 
+func GetLabels[O object](o O) map[string]string {
+	if md := o.GetMetadata(); md != nil {
```

Comment:
```suggestion
	if md := o.GetLabels(); md != nil {
```

Reply:
```
type object interface {
	GetMetadata() *meta.ObjectMeta
}
```

Are you saying to add `GetLabels` and `GetAnnotations` methods to the `object` interface?